conf/application.conf

```diff
@@ -136,8 +136,8 @@ microservice {
       protocol = http
       host = localhost
       port = 9491
```

Comment:
I think this port should be 9516 (if this is the "NCTS" service from service-manager-config)?

Reply:
sorry missed that one, updated now.